package.json

```diff
@@ -180,6 +180,9 @@
     ],
     "snapshotSerializers": [
       "jest-serializer-html"
+    ],
+    "setupFiles": [
+      "./setupTest.js"
```

Comment:
if you call it setupTests.js and you move it to src/ folder .  CRA will find it automatically so you don't have to add it here.

Reply:
ok